supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -510,19 +521,19 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     errorMessage.substring(0, errorMessage.indexOf(':')).toInt
   }
 
-  private def preempted(errorCode: Int, errorMessage: Option[String]): Boolean = {
+  private def preempted(errorCode: Int, errorMessage: List[String]): Boolean = {
```

Comment:
The return from the batch API gives you a list of errors instead of an option of one (in the case where the REST query fails, as opposed to the query successfully returning a failure message)